configuration/stub-frontend-fed-config/idp-rules-variant-c/stub-idp-no-docs.yml

```diff
@@ -0,0 +1,12 @@
+simpleIds: [ "stub-idp-no-docs" ]
```

Comment:
You want to look at https://github.com/alphagov/verify-frontend-federation-config/compare/HUB-737-tear-down-shorthub-abtest-and-keep-variant to adjust the rules to match

Reply:
Generally this is good stuff, and would probably just need a few tweaks once the remove short-hub is done.

Reply:
Yeah (see other comment) I agree... There is either a follow up ticket or we land this so you and John can make the changes as part of your ticket.  I am flexible either way.  I am keen to land this as it is now so this can stop being broken.  It might also help you and John with this now working.